index.html

```diff
@@ -2,53 +2,26 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Nelson Mandela Tribute Page</title>
     <link rel="stylesheet" href="css/style.css">
-    <style>
-      html{
-        font-size: 20px;
-      }
-      body{
-        font-family: 'Times New Roman', Times, serif;
-        font-weight: bold;
-      }
-      .center{
-        display: block;
-        margin-left: auto;
-        margin-right: auto;
-        width: 50%;
-      }
-      .nel{
-        display: block;
-        margin-left: auto;
-        margin-right: auto;
-        text-align: center;
-      }
-      .des{
-        font-size: medium;
-        font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
-        font-weight: lighter;text-align: center;
-
-      }
-    </style>
   </head>
 
   <body>
-    <main id= "main">
+    <main id="main">
       <h1 id="title" class="nel">Dr. Nelson Mandela</h1>
       <p class="des">A man with honor</p> 
-    <figure>
-      <id="img-div">
-      <img id="image" class="center" image src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/02/Nelson_Mandela_1994.jpg/220px-Nelson_Mandela_1994.jpg" 
-      alt="img-caption">Nelson Mandela</id>
-      <figcaption>Nelson Mandela in his office during his presidency</figcaption>
+    <figure class="img-div">
+      <img class="center" src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/02/Nelson_Mandela_1994.jpg/220px-Nelson_Mandela_1994.jpg" 
+      alt="Nelson Mandela in his day to day activities"/>
+      <figcaption class="img-caption">Nelson Mandela in his office during his presidency</figcaption>
     </figure>
-    <section id= "tribute info">
-      <h3 id="headline">This is Nelson Mandela's tribute</h3>
+    <section class="tribute info">
+      <h3 class="headline">This is Nelson Mandela's tribute</h3>
       <h4>
-        <p>Nelson Rolihlahla Mandela (18 July 1918 – 5 December 2013) was a South African anti-apartheid revolutionary political leader who served as the first president of South Africa from 1994 to 1999</p>
+      Nelson Rolihlahla Mandela (18 July 1918 – 5 December 2013) was a South African anti-apartheid revolutionary political leader who served as the first president of South Africa from 1994 to 1999
       </h4>
```

Comment:
Good job with this section again. However, I'd suggest checking the required design to ensure they look the same.
In this part, you could center your it and use the **ul** tag to group the items and the **li** tag to wrap each item. Additionally, I'd suggest adding more information with **li** tag wraps.